From patchwork Thu Jul 31 21:36:43 2025
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Brandon Brnich <b-brnich@ti.com>
X-Patchwork-Id: 26855
X-Patchwork-Delegate: vigneshr@ti.com
From: Brandon Brnich <b-brnich@ti.com>
Subject: [tiL6.12 v1 1/2] PENDING: arm64: dts: ti: k3-j722s-evm: Remove
 Improper Interrupt from main_pmx0
Date: Thu, 31 Jul 2025 16:36:43 -0500
Message-ID: <20250731213644.2168308-2-b-brnich@ti.com>
In-Reply-To: <20250731213644.2168308-1-b-brnich@ti.com>
References: <20250731213644.2168308-1-b-brnich@ti.com>
Return-Path: nm@ti.com
List-Post: <mailto:linux-patch-review@list.ti.com>
MIME-Version: 1.0
To: linux-patch-review@list.ti.com
Cc: Praneeth Bajjuri <praneeth@ti.com>, Devarsh Thakkar <devarsht@ti.com>,
 Udit Kumar <u-kumar1@ti.com>, Darren Etheridge <detheridge@ti.com>
List-ID: <linux-patch-review@list.ti.com>
List-Post: <mailto:linux-patch-review@list.ti.com>

main_pmx0 node in J722s inherits properties from same node in the am62p
dt. Interrupt was introduced in am62p dt for IO daisy chain wake up.
There is no IP that needs this interrupt on am62p. However, J722s uses
this interrupt line for the IMG E5010 JPEG Encoder.

Signed-off-by: Brandon Brnich <b-brnich@ti.com>
---
 arch/arm64/boot/dts/ti/k3-j722s-evm.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/ti/k3-j722s-evm.dts b/arch/arm64/boot/dts/ti/k3-j722s-evm.dts
index c66895b3f765..90226f06ba61 100644
--- a/arch/arm64/boot/dts/ti/k3-j722s-evm.dts
+++ b/arch/arm64/boot/dts/ti/k3-j722s-evm.dts
@@ -297,6 +297,7 @@ dp0_connector_in: endpoint {
 
 &main_pmx0 {
 
+	/delete-property/ interrupts;
 	main_mcan0_pins_default: main-mcan0-default-pins {
 		pinctrl-single,pins = <
 			J722S_IOPAD(0x1dc, PIN_INPUT, 0) /* (C22) MCAN0_RX */
